DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/vlan_utils.py

```diff
@@ -0,0 +1,51 @@
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+
+async def configure_bridge_setup(device, dut_ports, default_pvid=0):
+    device.applog.info("Creating bridge entity and enslaving links")
+    out = await IpLink.add(input_data=[
+        {device: [{"device": "br0", "type": "bridge", "vlan_filtering": 1, "vlan_default_pvid": default_pvid}]}])
```

Comment:
You are creating the bridge, but you are never removing it. It is better to create a fixture as Yurii did here https://github.com/PLVision/dent-testing/pull/18. And then maybe put it into `test/tast_suite/functional/vlan/conftest.py`

Reply:
isn't [these  ](here https://github.com/PLVision/dent-testing/pull/18.)fixtures  will be applied on teardown to all tests?

Reply:
Or @SerhiyBoikoPLV you mean to use fixtures from  that [PR ](https://github.com/PLVision/dent-testing/pull/18)

Reply:
You can add your own fixtures. Here is an example of how I did it for my mtu tests: [https://github.com/PLVision/dent-testing/pull/17](https://github.com/PLVision/dent-testing/pull/17/files#diff-c566038232def064b375e212681c4ccfcbe0e1340c537d882dc9927a7e816d32)